servers/physics_2d/godot_shape_2d.cpp

```diff
@@ -373,8 +373,7 @@ void GodotCapsuleShape2D::get_supports(const Vector2 &p_normal, Vector2 *r_suppo
 	if (h > 0 && Math::abs(n.x) > segment_is_valid_support_threshold) {
 		// make it flat
 		n.y = 0.0;
-		n.normalize();
-		n *= radius;
+		n.x = radius;
```

Comment:
This would change the behavior if `n.x` used to be negative. Is that a valid case that can happen?

I agree that the old code is bad though.

Reply:
You are right, the fix would be 
```
n.x = Math::sign(n.x) * radius;
```

Reply:
Or
```
n.x = n.x > 0 ? radius : -radius;
```

Reply:
The top makes more sense to me. Also, I think it preserves the behavior when `n.x` is zero.

Reply:
I wrongly assumed there was a `Math::sign` its a macro which is `SIGN` still changed it to use it